### ISSUE TITLE
IMPORTANT: issues with sf >= 0.9.0

### ISSUE THREAD

**Issue by ranghetti** [ranghetti/sen2r#295]
The recent update of package **sf** to version 0.9-0 is causing some issues to **sen2r** routines. Some of them were fixed in **sen2r** GitHub version (>= 1.3.2.9002), which can be installed with the following command:
```r
remotes::install_github("ranghetti/sen2r")
```

Some others are related to an incompatibility between **sf** >= 0.9-0 and **stars** (`sf::st_crs(stars::read_stars("/path/of/a/raster.file"))` is returning NA; see [CRAN results](https://cran.r-project.org/web/checks/check_results_stars.html)).

Until this incompatibility will be solved by **stars** maintainers, _please avoid upgrading **sf** to version 0.9-0_.
If you already updated it, you can downgrade it with the following command:
```r
remotes::install_version("sf", version = "0.8-1")
```

**Comment by ranghetti** [comment]
The above-cited commits _should_ have bypassed the existing problems. Nevertheless, I still recommend to keep **`sf`** version 0.8.1 until the incompatibility will be fixed (otherwise, some conflicts external to **`sen2r`** could remain within user scripts).
I will close this issue when this will be done by **`stars`** maintainers.

**Comment by ranghetti** [comment]
Problem was solved with the release of `stars` 0.4-1 on CRAN; packages can safely be updated.
